lang/en.json

```diff
@@ -1583,6 +1583,34 @@
     "written": "Written",
     "yard": "yard",
     "yards": "yards",
-    "youDoNotHavePermssion": "You do not have permission to do that."
+    "youDoNotHavePermssion": "You do not have permission to do that.",
+    "Dialog": {
+      "Save": "Save",
+      "Overwrite": "Overwrite"
+    },
+    "Importer": {
+      "Prompt": {
+        "Title": "Import Character Data for {name}",
+        "Description": "Select a file exported from GCS 5+.",
+        "Source": "Source Data",
+        "Note": "This import will overwrite the data for: <strong>{name}</strong>.",
+        "Warning": "<span class='fas fa-exclamation-triangle'></span>&nbsp;<strong>Note</strong>: This action cannot be undone!"
+      },
+      "PromptHPandFP": {
```

Comment:
If the setting is the same as one of the legacy settings, remove the old language file entries.